### PR TITLE
Update bardator.py

### PR DESCRIPTION
**Pull request by cbertoni**

### CODE REVIEW

**Comment by joseolapic** [comment]
:shipit: :trollface:

**Comment by cbertoni** [comment]
Quien tiene write access aca?

**Comment by toote** [comment]
:ship: :bowling:

**Comment by toote** [comment]
@cbertoni : mirá la url del repo

File: lalita/plugins/bardator.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def init(self, config):
 			u"SLM": u"SACATE LA MOCHILA!!! http://k32.kn3.net/taringa/F/0/F/F/4/1/agus1023/1FB.jpg",
 			u"IIHH": u"IIIIIHHH https://pbs.twimg.com/media/BnVsPkOCAAAomZ_.jpg",
 			u"LS": u"BYE BYE LITTLE SEBASTIAN!!!! https://www.youtube.com/watch?v=h-PUW6y4F6c",
+			u"BOWLING": u"ME HACEN BOWLING!!! http://i.embed.ly/1/display/resize?key=1e6a1a1efdb011df84894040444cdc60&url=http%3A%2F%2Fpbs.twimg.com%2Fmedia%2FBmmXma_IMAAvPy8.jpg",
```
Comment (xeBuz) [inline lalita/plugins/bardator.py]: ```http://pbs.twimg.com/media/BmmXma_IMAAvPy8.jpg```

**Comment by xeBuz** [comment]
:shipit: after comment (?)

**Pull request merged**